napari/layers/image/tests/test_pyramid.py

```diff
@@ -32,6 +33,63 @@ def test_infer_pyramid():
     assert layer._data_view.ndim == 2
 
 
+def test_infer_tuple_pyramid():
+    """Test instantiating Image layer with random 2D pyramid data."""
+    shapes = [(40, 20), (20, 10), (10, 5)]
+    np.random.seed(0)
+    data = tuple(np.random.random(s) for s in shapes)
+    layer = Image(data)
+    assert layer.data == data
+    assert layer.is_pyramid == True
+    assert layer.ndim == len(shapes[0])
+    assert layer.shape == shapes[0]
+    assert layer.rgb == False
+    assert layer._data_view.ndim == 2
+
+
+def test_forcing_pyramid():
+    """Test instantiating Image layer forcing 2D pyramid data."""
```

Comment:
This particular test is confusing to me... `is_pyramid=True` has no effect here? The resulting layer tests are the same for this test as for the one following it...

Reply:
the `assert layer.is_pyramid == True` is different, but I've also added some extra asserts about the `layer._data_pyramid` property that distinguish the True vs False state, the other properties were the same